cax/tasks/clear.py

```diff
@@ -183,3 +183,43 @@ def local_data_finder(self, thishost, version):
         return have_processed
 
 
+class PurgeProcessed(checksum.CompareChecksums):
+    """
+    Purge Processed root files
+    """
+
+    # Do not overload this routine from checksum inheritance.
+    each_run = Task.each_run
+
+    def each_location(self, data_doc):
+        """
+        Check every location with data whether it should be purged.
+        """
+        self.log.debug("Checking purge logic")
+
+        # Skip places where we can't locally access data
+        if 'host' not in data_doc or data_doc['host'] != config.get_hostname():
+            return
+
+        # See if purge settings specified, otherwise don't purge
+        if not config.purge_version() or (config.purge_version() == None) :
+            self.log.warning("Please set a pax version: vx.x.x")
```

Comment:
I thought the idea was: if you do not want to purge, then do not set the purge_version? 

In this case, the message should not be asking the user to set the version, since they actually do not want to purge.